src/terrain/clients/async_tasks.clj

```diff
@@ -0,0 +1,14 @@
+(ns terrain.clients.async-tasks
+  (:require [async-tasks-client.core :as async-tasks-client]
+            [clojure.string :as string]
+            [terrain.util.config :as config]))
+
+(defn run-async-thread
+  [async-task-id thread-function prefix]
+  (let [^Runnable task-thread (fn [] (thread-function))]
+    (.start (Thread. task-thread (str prefix "-" async-task-id))))
+  async-task-id)
```

Comment:
This was pretty much just copied out of `data-info`:
https://github.com/cyverse-de/data-info/blob/ebc288d08b3b6fa0df17008d04959bc1d7675535/src/data_info/clients/async_tasks.clj#L38-L42